io-engine/src/bin/io-engine-client/v1/stats_cli.rs

```diff
@@ -38,13 +47,14 @@ pub async fn handler(ctx: Context, matches: &ArgMatches) -> crate::Result<()> {
     }
 }
 
-async fn pool(mut ctx: Context, _matches: &ArgMatches) -> crate::Result<()> {
+async fn pool(mut ctx: Context, matches: &ArgMatches) -> crate::Result<()> {
     ctx.v2("Requesting Pool metrics");
+    let n = matches.get_one::<String>("name");
```

Comment:
nitpick: pool_name can be more appropriate here in place of `n`.
```suggestion
    let pool_name = matches.get_one::<String>("name");
```

Reply:
Done